hooks/useBlock.tsx

```diff
@@ -0,0 +1,24 @@
+import { useContext, useEffect } from 'react'
+
+import { BlockContext } from 'contexts/ServiceContexts'
+import { BlockType } from 'types'
+import useAsyncDataWrapper from './useAsyncDataWrapper'
+
+const useBlock = (id: string) => {
+  const service = useContext(BlockContext)
+  const [result, wrapper] = useAsyncDataWrapper<BlockType>()
+
+  useEffect(() => {
+    if (id) {
+      const options = isNaN(Number(id))
+        ? { hash: id, with_transactions: true }
+        : { sequence: Number(id), with_transactions: true }
```

Comment:
Do we want to support both `/blocks/<sequence>` and `/blocks/<hash>`?

Reply:
Yep.